src/components/SsoProviderSelector/SsoProviderSelector.tsx

```diff
@@ -0,0 +1,42 @@
+import type { FC } from "react";
+import EntryIdLogo from "./assets/EntryIdLogo";
+import KeycloakLogo from "./assets/KeycloakLogo";
+import OryHydraLogo from "./assets/OryHydraLogo";
+import Auth0Logo from "./assets/Auth0Logo";
+import SsoProviderButton from "./SsoProviderButton";
+
+const SsoProviderSelector: FC = () => {
+  return (
+    <div className="sso-provider-selector u-flex-column">
+      <p>Choose an SSO provider</p>
```

Comment:
In the modal case, we can probably raise this as the modal title and remove it here. Similar to the image below.

<img width="607" height="299" alt="image" src="https://github.com/user-attachments/assets/98ea0bfa-2e3f-4d3b-9774-665d266b3a3d" />